crates/taplo-cli/src/commands/format.rs

```diff
@@ -61,6 +62,10 @@ impl<E: Environment> Taplo<E> {
 
     #[tracing::instrument(level = "debug", skip_all)]
     async fn format_files(&mut self, cmd: FormatCommand) -> Result<(), anyhow::Error> {
+        if cmd.stdin_filepath.is_some() {
+            bail!("`--stdin-filepath` can only be used with stdin");
```

Comment:
Bailing here feels a bit too much, the flag just rewrites `-` to be whatever, maybe we could just ignore it here or probably print a warning?

Reply:
Yes, I totally agree with you here. Thanks.